CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v6.2.0-rc1 (07/22/2021)
+
+#### Features:
+
+* FlashArray NVMeoF-RoCE support (NVMeoF using RDMA over Converged Ethernet)
+* A fresh PSO installation is recommended to use this release only if you want to try the NVMeoF-RoCE feature.
```

Comment:
I would flip this sentence order, and then upgrade it to a "must" (I don't think I'd trust moving a cluster from iSCSI or FC to NVMe in place).
```suggestion
* If you want to try the NVMeoF-RoCE feature, a fresh PSO installation is required.
```